diff --git a/kernel/sys_config/hi3516cv6xx/sys_cfg.c b/kernel/sys_config/hi3516cv6xx/sys_cfg.c
--- a/kernel/sys_config/hi3516cv6xx/sys_cfg.c
+++ b/kernel/sys_config/hi3516cv6xx/sys_cfg.c
@@ -27,3 +27,4 @@
 #define SENSOR_NAME_SC431HAI "sc431hai"
 #define SENSOR_NAME_OS04D10 "os04d10"
+#define SENSOR_NAME_IMX662 "imx662"
 #define MIPI_AD "mipi_ad"
@@ -286,8 +287,9 @@
     if ((strncmp(SENSOR_NAME_SC4336P, name, len) == 0) ||
         (strncmp(SENSOR_NAME_GC4023, name, len) == 0) ||
         (strncmp(SENSOR_NAME_SC431HAI, name, len) == 0) ||
         (strncmp(SENSOR_NAME_SC450AI, name, len) == 0) ||
-        (strncmp(SENSOR_NAME_SC500AI, name, len) == 0)) {
+        (strncmp(SENSOR_NAME_SC500AI, name, len) == 0) ||
+        (strncmp(SENSOR_NAME_IMX662, name, len) == 0)) {
         clock = 0xA001;   /* 27M */
     } else if ((strncmp(SENSOR_NAME_OS04D10, name, len) == 0)) {
         clock = 0x4001;  /* 24M */
